suite-native/device/src/hooks/usePinAction.tsx

```diff
@@ -110,15 +110,21 @@ export const usePinAction = ({ type, onSuccess }: PinActionProps) => {
             onSuccess();
         } else {
             const errorCode = payload.code;
-            if (errorCode === 'Failure_ActionCancelled' || errorCode === 'Failure_PinCancelled') {
+            if (
+                errorCode === 'Failure_ActionCancelled' ||
+                errorCode === 'Failure_PinCancelled' ||
+                errorCode === 'Method_Interrupted'
+            ) {
                 showError(canceledMessageKey, handlePinAction);
```

Comment:
Nitpick: canceled message says "has been canceled on your Trezor" but it doesn't have to be true (it appears if you cancel from the app too...). I would just remove the part "on your Trezor" from all the cancelled messages, but we can do it in next PR together with handling GO_BACK navigation on PIN screens.

Reply:
Yeah, this also could actually be an enhancement from @trezor/suite-connect  to be able to differentiate between a cancel action from the app and from the Trezor. 🤔?

Reply:
`Method_Interrupted` or `Method_Cancel` are the ones coming from the app

Reply:
I believe we can handle it but I don't believe it's worth it here...

Reply:
When I tried to debug it it showed the same message, regardless the origin.
So maybe the `pin cancel ` action is throwing a wrong msg?
 btw: https://github.com/trezor/trezor-suite/pull/18833/commits/ceec45ed9bc113ea1be1b0d2293700142423a2fe

Reply:
only T1 trhows `Method_Interrupted` thats why we needed the "fix"